sprint6/DO_NOT_TOUCH/zadanie3.js

```diff
@@ -0,0 +1,16 @@
+/*
+Zadanie 3. Napisz funkcję, która wypisze wszystkie liczby od 0 do 100
+*/
+
+function writeNumbersTo100() {
+  for (let i = 0; i <= 100; i++) {
+    console.log(i);
+  }
+}
+
+/*
+Stworzyłam funkcję o nazwie "writeNumbersTo100". W funkcji ustawiam licznik
+na wartość zerową, następnie ustalam, ze pętla zakończy się gdy nasz licznik
+dojdzie do 100. Wypisywana wartość pętli zaczyna się od zera i jest zwiększana
+o jeden az osiągnie liczbę 100. W pętli wartość wypisuje kazdej liczbie całkowitej.
+*/
```

Comment:
`Wypisywana wartość pętli zaczyna się od zera i jest zwiększana o jeden az osiągnie liczbę 100. W pętli wartość wypisuje kazdej liczbie całkowitej.` - To wszystko proszę wywalić. Zamiast tego napisac zdanie typu: `W pętli wykonywane jest ...`